### PR TITLE
Test doc updates

### LINKED ISSUE #333: Test plan for existing VCH Management UI functionality
**Details:**
For 1.3.0, an initial set of functionality was implemented for the VCH Management UI including creation and deletion. Testing of that functionality was not systematic.

We should step back and product a comprehensive test plan, covering both unit and integration tests, to ensure we have a solid foundation as we refactor and enhance.

**Acceptance Criteria:**

- [ ] Test plan summarizes existing functionality.
- [ ] Test plan explains how to test existing functionality.
- [ ] Test plan identifies gaps where additional tests are needed (including discussion of late-breaking bugs found in 1.3).
- [ ] Issues are open and prioritized for identified gaps.

### LINKED ISSUE #363: Test plan for 1.4 VIC UI functionality
Details:
For 1.4 proposed UI features, produce and initial test plan, covering both unit and integration tests, to ensure we have a solid foundation for new 1.4 features.

Current proposed features include:
VIC Upgrade
Edit VCH

Acceptance Criteria:
 Test plan summarizes required functionality.
 Test plan explains how to test required functionality.
 List of initial test groups and cases

### PR DESCRIPTION
**Pull request by jak-atx**
Fixes #363 
Fixes #333 

* Added README.md with testing overview
* Updated TestCases.md 
* Additional test docs for create 
* Initial test docs for upgrade
* Rename of manual-test-cases to test-cases

PR acceptance checklist:

[X] Build with unit tests pass
[n/a] All e2e tests pass
[n/a] Unit test(s) included
[n/a] e2e test(s) included
[n/a] Screenshot attached and UX approved

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
++# VIC UI Testing
```
Comment (jooskim) [inline tests/README.md]: Extra + signs seem to screw the MD syntax. Need to fix the formatting.

File: tests/test-cases/Group1-VCH-Creation-Wizard/1-02-Multi-DVS-VCH-Create.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
++Test 2-02 - Multi DVS VCH Create
```
Comment (jooskim) [inline tests/test-cases/Group1-VCH-Creation-Wizard/1-02-Multi-DVS-VCH-Create.md]: Extra + signs seem to screw the MD syntax. Need to fix the formatting.

File: tests/test-cases/Group2-VIC-Upgrade/2-01-VIC-Upgrade.robot [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# TBD
+# [ Windows 10 - Chrome ]
+# [ Windows 10 - IE11 ]
+# [ MacOS - Chrome ]
+# [ MacOS - Firefox ]
+# [ Windows 10 - Chrome ]
+# [ Windows 10 - Firefox ]
+# [ Windows 10 - IE11 ]
+# [ MacOS - Chrome ]
+# [ MacOS - Firefox ]
```
Comment (jooskim) [inline tests/test-cases/Group2-VIC-Upgrade/2-01-VIC-Upgrade.robot]: New line needed here

**Comment by jak-atx** [comment]
Jenkins job will need to be updated with the new paths once this is merged

**Pull request merged**